dolphinscheduler-ui-next/README.md

```diff
@@ -1,14 +1,50 @@
-### Dolphin Scheduler UI Next
+<h1 align="center">Dolphin Scheduler UI Next</h1>
```

Comment:
Why do we need these? please refer https://github.com/apache/dolphinscheduler-website/pull/615